src/templates/_common/components/elements.jsx

```diff
@@ -0,0 +1,187 @@
+import React from 'react'; // eslint-disable-line
+
+export const Li = ({
+    className,
+    id,
+    href,
+    param = '',
+    target,
+    text,
+    header,
+    p,
+}) => {
+    const content = p ? <p> {text} </p> : text;
```

Comment:
we should remove the white space between `<p>` and `{text}` (?)

Reply:
@negar-binary Yea, it's a styling improvement ...however the functionality will remain intact.

Reply:
are you sure? as we tested with @binary-hamedanchi it keeps the space in the generated template

Reply:
@negarn Does the browser care about the whitespace?

Reply:
I think sometimes, yes, but if we want to rely on it handling most cases then you can leave as is